Makefile

```diff
@@ -186,14 +186,18 @@ stylesheets: public/css/index.css
 
 .IGNORE: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
-	lessc $< $@
+	@hash lessc > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		docker run --rm -v $(PWD):/app appleboy/node-less $< $@; \
+	else \
+		lessc $< $@; \
+	fi
```

Comment:
In place of docker we could try https://github.com/kib357/less-go. It has a cli interface ready under : github.com/kib357/less-go/lessc. It is a little hacky implementation but it should work.